core/dbt/cli/main.py

```diff
@@ -297,7 +297,7 @@ def debug(ctx, **kwargs):
 @p.target
 @p.vars
 @requires.preflight
-@requires.profile
+@requires.unset_profile
```

Comment:
I like it!

Eventual goal is, we remove the need for this dummy profile entirely? (https://github.com/dbt-labs/dbt-core/issues/6257)

Reply:
yep exactly! I'll leave a comment.